templates/github/.ci/ansible/settings.py.j2.copy

```diff
@@ -8,7 +8,6 @@ TOKEN_SIGNATURE_ALGORITHM = "ES256"
 CACHE_ENABLED = True
 REDIS_HOST = "localhost"
 REDIS_PORT = 6379
-TELEMETRY = False
```

Comment:
Don't remove that, replace it by ANALYTICS please.

Reply:
will do.